src/main/java/wooteco/chess/controller/SparkChessController.java

```diff
@@ -0,0 +1,84 @@
+package wooteco.chess.controller;
+
+import static spark.Spark.*;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+import spark.ModelAndView;
+import spark.Request;
+import spark.Response;
+import spark.template.handlebars.HandlebarsTemplateEngine;
+import wooteco.chess.service.ChessService;
+import wooteco.chess.service.dto.ChessBoardDto;
+import wooteco.chess.service.dto.ChessGameDto;
+import wooteco.chess.service.dto.ChessStatusDtos;
+
+public class SparkChessController {
+
+	private final ChessService chessService;
+
+	public SparkChessController(final ChessService chessService) {
+		Objects.requireNonNull(chessService, "체스 서비스가 null입니다.");
+		this.chessService = chessService;
+	}
+
+	private static String render(Map<String, Object> model, String templatePath) {
+		return new HandlebarsTemplateEngine().render(new ModelAndView(model, templatePath));
```

Comment:
제가 받았던 리뷰 내용입니다.

HandlebarsTemplateEngine 객체를 반복해서 생성하는 것은 굉장히 비용이 큰 작업입니다.
재사용할 수 있도록 어떠한 장점이 있을지 고민해보아요.

충분한 고민 후 이펙티브 자바 아이템 6. 불필요한 객체 생성을 피하라 참고해주세요!

이에 대해서 제가 수정했던 방안은 
private static final HandlebarsTemplateEngine handlebarsTemplateEngine = new HandlebarsTemplateEngine();
클래스 field로 생성후
```suggestion
	             return handlebarsTemplateEngine.render(new ModelAndView(model, templatePath));
```
이렇게 수정했는데요! 한번 고려해보셔도 좋을듯 합니다!